Tests/PointFreeTests/AccountTests/__Snapshots__/ChangeTests/testChangeShowDiscountSubscription.1.Conn.txt

```diff
@@ -1846,7 +1846,7 @@ for (var idx = 0; idx < elements.length; idx++) {
                            ts-d-r1_5
                            ts-m-r1_25
                            lh-2">
-                  Change to yearly billing?
+                  Change to monthly billing?
```

Comment:
this snapshot was technically wrong since the mock used for it used a yearly plan, but the `interval` in that plan was set to `monthly`. so this is the correct one now.